packages/fxa-auth-server/config/index.ts

```diff
@@ -238,12 +238,6 @@ const conf = convict({
       env: 'AMPLITUDE_SCHEMA_VALIDATION',
       format: Boolean,
     },
-    rawEvents: {
```

Comment:
There's still some references to this in mocks in tests. Search code for 'rawEvents'.